src/main/java/org/zeromq/ZMQException.java

```diff
@@ -4,17 +4,8 @@
 
 public class ZMQException extends RuntimeException
 {
-    public static class IOException extends RuntimeException
-    {
-        private static final long serialVersionUID = 8440355423370109164L;
-
-        public IOException(java.io.IOException cause)
-        {
-            super(cause);
-        }
-    }
 
-    private static final long serialVersionUID = 5957233088499712341L;
+    private static final long serialVersionUID = -978820750094924644L;
```

Comment:
Good explanation here:
http://stackoverflow.com/questions/285793/what-is-a-serialversionuid-and-why-should-i-use-it/285809